std/math.d

```diff
@@ -3927,6 +3927,91 @@ float floor(float x) @trusted pure nothrow @nogc
     assert(isNaN(floor(float.init)));
 }
 
+/**
+ * Round `val` to a multiple of `unit`. `rfunc` specifies the rounding
+ * function to use; by default this is `rint`, which uses the current
+ * rounding mode.
+ */
+real quantize(alias rfunc = rint)(real val, real unit)
```

Comment:
Don't we also want `double` and `float`? How about a generic with `isFloatingPoint`?